src/shared/components/cohort/FrequencyBar.tsx

```diff
@@ -63,7 +74,7 @@ export default class FrequencyBar extends React.Component<IFrequencyBarProps, {}
                     textAnchor="start"
                     fontSize="10"
                 >
-                    {`${(100 * mainProportion).toFixed(1)}%`}
+                    {getPercentage(mainProportion)}
```

Comment:
For consistency, using `getPercentage` function for the frequency bar as well...